packages/ffe-cards-react/src/index.d.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+export interface CardBaseProps extends React.HTMLAttributes {
+    children: React.ReactNode;
+    className?: string;
+    element?: React.ReactNode;
+    to?: string;
+}
```

Comment:
```suggestion
export interface CardBaseProps extends ComponentBaseProps {
    to?: string;
}
```
Trying to use suggestions here - not sure if it will work properly (?). My point here is however, can't you just extend ``CardBaseProps`` in ``ComponentBaseProps``? I see there is a difference in "element", not sure if that's intentional? :thinking: 

Also check out my suggestion for removing ``ComponentBaseProps`` further down in this file.

Reply:
No that's just an oversight, we definetely don't want react-nodes in `element` since that can include numbers and lots of other things.